Classes/Controller/PageViewController.php

```diff
@@ -217,30 +602,54 @@ protected function getAnnotationContainers(int $page): array
      *
      * @param int $page Page number
      *
+     * @param ?AbstractDocument $specificDoc
```

Comment:
Type `AbstractDocument` seems to be wrong here, because the method `getCurrentDocument` is called on this object and this doesn't exist for `AbstractDocument`. Could you please check, what type `$specificDoc` is?

This should fix the following `Call to an undefined method` errors.